app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] Dear Team, Good job setting up the Devise for your app. but we need to make sure the application asks for confirmation of email. and for this, we need to enable the `:confirmable` module here.